src/devtools/client/debugger/src/components/Editor/ColumnBreakpoints.tsx

```diff
@@ -26,33 +29,58 @@ type PropsFromRedux = ConnectedProps<typeof connector>;
 
 type $FixTypeLater = any;
 interface CBProps {
-  editor: $FixTypeLater;
+  editor: SourceEditor;
 }
 
-class ColumnBreakpoints extends Component<PropsFromRedux & CBProps> {
-  render() {
-    const { cx, editor, columnBreakpoints, selectedSource } = this.props;
-
-    if (!selectedSource || columnBreakpoints.length === 0) {
-      return null;
-    }
-
-    let breakpoints;
-    // TODO Is this a safe thing to do while rendering?
-    editor.codeMirror.operation(() => {
```

Comment:
:speech_balloon: I really have no idea why this `cm.operation()` call was here in the first place. We're mapping over data.

Only thing I can figure is that maybe at some point early on we were inserting widgets directly into CM?

Reply:
Yeah, I had the same thought (and removed it) as part of the points refactor branch.